packages/byods-demo-server/src/server.ts

```diff
@@ -0,0 +1,183 @@
+import express, {Request, Response, NextFunction} from 'express';
+import logger from 'jet-logger';
+import * as BYODS from '@webex/byods/src/index';
+import 'express-async-errors';
+import path from 'path';
+import helmet from 'helmet';
+import morgan from 'morgan';
+import cookieParser from 'cookie-parser';
+import handlebars from 'handlebars';
+import fs from 'fs';
+
+// **** Core Setup **** //
+const app = express();
```

Comment:
**Note**
The code here is rough and needs some cleanup. It is functional, however I am working on making it more readable and follow proper design principles. 
Open to suggestions however.